change/@ni-nimble-components-063e914a-8704-466c-b58f-6ad350207e57.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Change type can be `none` since this doesn't impact the published package in any way.
```suggestion
  "type": "none",
```